doc/source/release-notes/Percona-Server-5.7.17-11.rst

```diff
@@ -0,0 +1,94 @@
+.. rn:: 5.7.17-11
+
+==========================
+|Percona Server| 5.7.17-11
+==========================
+
+Percona is glad to announce the GA (Generally Available) release of |Percona
+Server| 5.7.17-11 on February 3rd, 2017 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.17-11/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.7.17
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-17.html>`_, including
+all the bug fixes in it, |Percona Server| 5.7.17-11 is the current GA release
+in the |Percona Server| 5.7 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.7.17-11
+milestone at
+Launchpad <https://launchpad.net/percona-server/+milestone/5.7.17-11>`_
+
+New Features
+============
+
+ |Percona Server| has implemented support for per-column ``VARCHAR/BLOB``
+ :ref:`compression <compressed_columns>` for the |XtraDB| storage engine. This
+ also features compression dictionary support, to improve compression ratio for
+ relatively short individual rows, such as JSON data.
+
+ :ref:`innodb_kill_idle_trx` feature has been re-implemented by setting a
+ different socket read timeout value instead of periodically scanning the low
+ |InnoDB| transaction list level. This re-implementation is also addressing
+ some existing bugs: :bug:`1166744`, :bug:`1179136`, :bug:`907719`, and
+ :bug:`1369373`.
+
+Bugs Fixed
+==========
+
+ Logical row for |TokuDB| tables could get inaccurate over time. Bug fixed
+ :bug:`1651844` (:ftbug:`732`).
+
+ Repeated execution of ``SET STATEMENT ... FOR`` could lead to a server crash.
+ Bug fixed :bug:`1392375`.
+
+ ``CREATE TEMPORARY TABLE`` would create a transaction in binary log on read
+ only server. Bug fixed :bug:`1539504`.
+
+ Using :ref:`per_query_variable_statement` with subquery temporary tables could
+ cause a memory leak. Bug fixed :bug:`1635927`.
+
+ Fixed new compilation warnings with GCC 6. Bugs fixed :bug:`1641612` and
+ :bug:`1644183`.
+
+ If a bitmap write I/O errors happened in the background log tracking thread
```

Comment:
A server could crash if a bitmap write I/O error happens in the background log tracking thread while a FLUSH CPB is executing concurrently.